docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 ## March 2024
 
+**13 March 2024**
+
+Frontend version: 1.0.0-20240306.1552 | Backend version: 1.95.14-20240311.1204
+
+| Type | Change| Description|
+ ------ | ------- | ----------- |
+| **Feature** | Expose SAML features | All SAML apps are now appropriately tagged with the correct namespace and owners, ensuring their visibility on the portal. |
```

Comment:
don't need to mention 'tagged with the correct namespace and owners', cuz it's an internal operation that tenants don't need to be aware of

also 'ensuring their visibility on the portal' sounds a bit vague, can mention they can now manage SAML apps on the portal